mu-plugins/blocks/sidebar-container/src/view.js

```diff
@@ -1,19 +1,16 @@
 /**
  * Fallback values for custom properties match CSS defaults.
  */
-const globalNavHeight = 90;
 
-const LOCAL_NAV_HEIGHT = getCustomPropValue( '--wp--custom--local-navigation-bar--spacing--height' ) || 60;
+const GLOBAL_NAV_HEIGHT = getCustomPropValue( '--wp-global-header-height' ) || 90;
```

Comment:
Making this dynamic fixes a bug where top and bottom positions are incorrect, caused by the global header being shorter on short screens.